client/src/pages/Login/Login.tsx

```diff
@@ -36,10 +37,30 @@ export default function Login(): JSX.Element {
     });
   };
 
+  const demoLogin = () => {
+    const email = 'demo@demo.com';
+    const password = 'test12345';
+    login(email, password).then((data) => {
+      if (data.error) {
+        updateSnackBarMessage(data.error.message);
+      } else if (data.success) {
+        console.log('data---', data);
+        updateLoginContext(data.success);
+      } else {
+        // should not get here from backend but this catch is for an unknown issue
+        console.error({ data });
+        updateSnackBarMessage('An unexpected error occurred. Please try again');
+      }
```

Comment:
Is there ever a case where we aren't looking for data.error or data.success to have a catchall  here? Would this even be reachable? 

My point is, if there is an error it would be on data.error or data.success?

Reply:
imrpoved error handling.